### ISSUE TITLE
Block Editor Paragraphs missing tags

### ISSUE THREAD

**Issue by sarahfg** [wordpress-mobile/WordPress-Android#11030]
### Expected behavior
When using the block editor in the app paragraphs created should have opening and closing HTML tags

### Actual behavior
When adding paragraphs using the android app's block editor `<p>` tags are not added. They are all in one `div` like this:

![](https://d.pr/i/uBQ2Uk+)

(Image link: https://d.pr/i/uBQ2Uk )

### Steps to reproduce the behavior
1. Create a new WordPress site
2. Log into the new site in the WordPress Android app
3. Enable the block editor
4. Create a new post or page using paragraphs
5. Publish the post or page

##### Tested on 
Pixel 3, Android 10, WPAndroid alpha-201
Pixel 3 XL, WPAndroid 13.8

To test I used a new WordPress.org site created at https://jurassic.ninja/ to test with no active plugins and the theme Twenty-Twenty active

User reported in #2581137-zen

**Comment by designsimply** [comment]
@sarahfg thank you for reporting this! I tested and confirmed that paragraph tags aren't appearing when using the block editor for self-hosted or Jetpack-connected sites while the same problem does not happen for WordPress.com simple sites.

I created an issue at https://github.com/wordpress-mobile/gutenberg-mobile/issues/1728 to track this in the `gutenberg-mobile` repo but will leave the issue open here as well since a `[Pri] Blocking` label was added. 👍
